From 4282525f652e79f1d98e58f8ef108547b7256f45 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lubom=C3=ADr=20Sedl=C3=A1=C5=99?= <lsedlar@redhat.com>
Date: Thu, 9 Nov 2017 14:11:41 +0100
Subject: [PATCH] gather: get_packages_to_gather returns a tuple
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It always needs to return a tuple, otherwise there would be crashes.

Signed-off-by: Lubomír Sedlář <lsedlar@redhat.com>
---
 pungi/phases/gather/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pungi/phases/gather/__init__.py b/pungi/phases/gather/__init__.py
index 3b96abc..c4b27f0 100644
--- a/pungi/phases/gather/__init__.py
+++ b/pungi/phases/gather/__init__.py
@@ -517,7 +517,7 @@ def get_packages_to_gather(compose, arch=None, variant=None, include_arch=True,
         be included in a list of packages.
     """
     if compose.conf["gather_source"] == "module":
-        return []
+        return ([], [])
 
     arches = [arch] if arch else compose.get_arches()
 
-- 
2.13.6

